sugoi-api-rest-services/src/main/java/fr/insee/sugoi/services/utils/CsvConverter.java

```diff
@@ -0,0 +1,49 @@
+package fr.insee.sugoi.services.utils;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class CsvConverter {
```

Comment:
Why not use a csv library? (commons-csv or other). There are some edge case around formatting csv which should be better managed.

Reply:
ok for common-csv